### DISCUSSION TITLE
White screen in the  Marketplaces page

### DISCUSSION THREAD

**Discussion by VasylDvorakDS** [FlowiseAI/Flowise#3186] [Q&A]
When I try to start Marketplaces in the localhost:3000 I receve only white screen in the browser Mazila and other browsers? How can I fix this problem, tell me please.

**Comment by JamelSaladin** [comment]
I am having the same issue and I have already updated everything, however I will say that I did not have this problem when I was running an older version of node.js (18), once I updated to like 22 then all of a sudden everytime I click on marketplaces the whites screen pops up or should I say the page opens and is all white.

**Reply by Alexis-mendez2494** [reply]
Which version are you using? version flowise@1.6.0 has this error. Try switching to version 2.1.3 using -> "npm i -g flowise@2.1.3"

**Comment by cedricfossito** [comment]
try to clear up your cache using this commande.     
npm cache clean --force

then update your flowise with this commande
npm install -g flowise@latest

**Comment by mikz0rz** [comment]
Workaround is to downgrade the Node version: https://github.com/FlowiseAI/Flowise/issues/3583
